src/components/Jobs/Loader/Loader.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types'
+import './loader.css'
+
+function Loader({ isSubmitted, jobData }) {
+  return (
+    <div className={`loading ${isSubmitted && jobData === false ? '' : 'hidden'}`}>
+      <div className="ball first"></div>
+      <div className="ball second"></div>
+      <div className="ball third"></div>
+    </div>
+  )
+}
+
+export default Loader
+
+Loader.propTypes = {
+  isSubmitted: PropTypes.bool,
+  jobData: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
```

Comment:
Is `jobData` a boolean or an object?

Reply:
fixed! it's now just an object, used your super handy '?' modifier to test if jobData was there instead of using true/false so two different data types aren't being passed to the same prop anymore.